frontend/public/components/markdown-view.tsx

```diff
@@ -89,13 +89,19 @@ export class SyncMarkdownView extends React.Component<
         <link rel="stylesheet" href="${link.href}">`,
       '',
     );
+    const content = this.props.truncateContent
+      ? _.truncate(this.props.content, {
+          length: 100,
```

Comment:
100 feels a little too short. Maybe 256?

Reply:
I wanted to leave it to approx. 3 lines, which is about 100+- chars

Reply:
> I wanted to leave it to approx. 3 lines, which is about 100+- chars

I don't think it's enough content to describe some resources. We're really trying to catch the extreme cases where there are paragraphs of text.